src/test/java/org/htmlunit/javascript/host/LocationTest.java

```diff
@@ -412,30 +415,30 @@ public void reloadWithHash() throws Exception {
         String date = page.getElementById("date").asNormalizedText();
         page = page.getElementById("reload").click();
         String newDate = page.getElementById("date").asNormalizedText();
-        assertEquals(date, newDate); // FIXME: this should be assertNotEquals
+        assertNotEquals(date, newDate);
```

Comment:
sorry could you change this to use `assertNotSame()` instead of `assertNotEqual()` and remove the 
```
import static org.junit.Assert.assertNotEquals;
```